scenario-compiler/src/main/scala/pl/touk/nussknacker/engine/spel/SpelExpressionTypingError.scala

```diff
@@ -2,13 +2,15 @@ package pl.touk.nussknacker.engine.spel
 
 import cats.data.NonEmptyList
 import org.springframework.expression.spel.SpelNode
-import pl.touk.nussknacker.engine.api.context.ProcessCompilationError
 import pl.touk.nussknacker.engine.api.generics.{ExpressionParseError, Signature}
-import pl.touk.nussknacker.engine.api.parameter.ParameterName
+import pl.touk.nussknacker.engine.api.generics.ExpressionParseError.TextErrorDetails
 import pl.touk.nussknacker.engine.api.typed.typing.{TypedDict, TypingResult}
+import pl.touk.nussknacker.engine.expression.PositionRange
+import pl.touk.nussknacker.engine.spel.SpelExpressionParseError.SpelExpressionTypingParseError
+import pl.touk.nussknacker.engine.spel.SpelExpressionTypingError.SpelExpressionTypingError
 
-object SpelExpressionParseError {
-  trait SelectionProjectionError extends ExpressionParseError
+object SpelExpressionTypingError {
```

Comment:
All errors returned in validation are not `ExpressionParseError` because we want to enrich error details (position range) in one place